### PR TITLE
fix: add min width/height so it doesn't break the design

### LINKED ISSUE #81: Set min size of the xud-explorer window
Off-shoot from #72 :

Should be done for electron app, in browser we'd solely want a "please increase window size" if window is below a certain threshold that breaks things.

### PR DESCRIPTION
**Pull request by scorebreaker**
Closes #81

### CODE REVIEW

**Review by raladev — CHANGES_REQUESTED** [review]
Del

File: public/electron.js [hunk 1]
```diff
@@ -11,8 +11,10 @@ let mainWindow;
 
 function createWindow() {
   mainWindow = new BrowserWindow({
-    width: 900,
     height: 680,
+    minHeight: 150,
```
Comment (raladev) [inline public/electron.js:15]: minHeight: 680

File: public/electron.js [hunk 2]
```diff
@@ -11,8 +11,10 @@ let mainWindow;
...
+    width: 900,
+    minWidth: 600,
```
Comment (raladev) [inline public/electron.js:17]: minWidth: 900

**Review by raladev — CHANGES_REQUESTED** [review]
window size lower then 900x600 breaks layout on certain pages.
minHeight may be reduced to 600, but there is no reason for this.

**Review by krrprr — CHANGES_REQUESTED** [review]
Let's set the min sizes same as the initial window size.
Otherwise looks good to me.

**Review by krrprr — APPROVED** [review]
Code-wise ACK, basic testing passed.
Thank you!

**Review by raladev — APPROVED** [review]

**Pull request merged**